frontend/src/console/components/PreviewComponent/PreviewComponent.tsx

```diff
@@ -18,38 +19,42 @@ export const PreviewComponent: React.FC<PreviewComponentProps> = (
   const { instanceData } = props;
   const themeData = instanceData.draftThemeConfig!;
 
-  return (
-    <div className="theme-preview">
-      <Row className="theme-preview-navigation">
-        <Col className="theme-preview-navigation">
-          <NavigationMenu instanceData={instanceData} themeData={themeData} />
-        </Col>
-      </Row>
-      <div className="theme-home">
-        <Row className="theme-hero-container">
-          <Col>
-            <HeroPreview
-              heroCoverImage={instanceData.heroCoverImage || ''}
-              homePageHeroTitleColor={themeData.homePageHeroTitleColor}
-              homePageHeroSubtitleColor={themeData.homePageHeroSubtitleColor}
-              homepageOverlayHtml={
-                instanceData!.draftStaticContentOverrides &&
-                instanceData.draftStaticContentOverrides.homepageOverlayHtml
-              }
-            />
+  // To Do: Edit this to taste when implementing the other previews
```

Comment:
If tickets to implement other previews are scheduled, this comment can be dropped.

Reply:
Agreed.